sw/device/lib/testing/test_rom/english_breakfast_fake_driver_funcs.c

```diff
@@ -0,0 +1,33 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <assert.h>
+
+#include "sw/device/silicon_creator/lib/drivers/lifecycle.h"
+#include "sw/device/silicon_creator/lib/drivers/otp.h"
+#include "sw/device/silicon_creator/lib/drivers/rstmgr.h"
+
+// TODO(#12905): This file includes fake definitions for the functions that are
+// used by the silicon_creator bootstrap implementation but missing in the
+// english breakfast top level due to hardware limitations.
+
+#if !OT_IS_ENGLISH_BREAKFAST
+static_assert(
```

Comment:
nit: Also `#error "error msg"` but I assume `static_assert` is more consistent with the codebase.

Reply:
Good point, done, thanks!